client/src/App.tsx

```diff
@@ -11,47 +12,12 @@ const queryClient = new QueryClient({
   },
 });
 
-const Wrapper = styled.div`
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  padding: 80px 0;
-`;
-
-const H1 = styled.h1`
-  font-weight: bold;
-  font-size: 40px;
-  margin-bottom: 64px;
-`;
-
-const H3 = styled.h3`
-  font-weight: bold;
-  font-size: 20px;
-  margin-bottom: 16px;
-`;
-
-const Img = styled.img`
-  width: 300px;
-  cursor: pointer;
-`;
-
-const onImageClick = () => {
-  window.location.href = 'https://www.chromatic.com/library?appId=611b598bab18bf003a128fbf';
-};
-
 const App = (): JSX.Element => {
   return (
     <QueryClientProvider client={queryClient}>
-      <Wrapper>
-        <H1>공사중입니다...</H1>
-        <H3>Storybook을 보시려면 아래 이미지를 눌러주세요.</H3>
-        <Img
-          onClick={onImageClick}
-          src="/images/bff24bd5f52a4d0e5ea5eaeb34bed508113e2bd2b7407c8202a97d2241a96625.png"
-          alt="bang-goo"
-        />
-      </Wrapper>
+      <Router>
+        <Route path="/" component={Main}></Route>
```

Comment:
```suggestion
        <Route path="/" component={Main} />
```

내부에 아무것도 없다면 바로 닫아주는 편이 좋을 것 같습니다! 😄

Reply:
자동완성이 되어서 직접 수정하느라 자꾸 놓치네요~~ ㅠㅠ